### PR TITLE
Issue #9120: Migrate ManifestUpdateFeature to use BrowserStore instead of SessionManager.

### PR DESCRIPTION
**Pull request by pocmo**
Together with PR #9117 this should complete the migration of `feature-pwa`.

### CODE REVIEW

**Comment by csadilek** [comment]
Test failure looks strange / like an IO problem:
```java.lang.NullPointerException: ParseSearchPluginsTest::…Resource("searchplugins") must not be null```

Let's try this again.

File: components/feature/search/build.gradle [hunk 1]
```diff
@@ -25,7 +25,7 @@ android {
             resources {
                 // We want to access the assets from unit tests. With this configuration we can just
                 // read the files directly and do not need to rely on Robolectric.
-                srcDir "${rootDir}/components/browser/search/src/main/assets/"
+                // srcDir "${rootDir}/components/browser/search/src/main/assets/"
```
Comment (pocmo) [inline components/feature/search/build.gradle:28]: Caused by this line, that I didn't want to push :)

File: components/feature/pwa/build.gradle [hunk 2]
```diff
@@ -63,6 +63,7 @@ dependencies {
     kapt Dependencies.androidx_room_compiler
 
     testImplementation project(':support-test')
+    testImplementation project(':support-test-libstate')
```
Comment (csadilek) [inline components/feature/pwa/build.gradle:67]: We should be able remote `browser-session` here now?

**Review by csadilek — APPROVED** [review]
One question below. Let's land the 3 PRs for the same Nightly on Monday so we only have breaking changes once :).

File: components/feature/pwa/build.gradle (hunk 2 above)
Comment (pocmo) [reply components/feature/pwa/build.gradle:67]: Yeah! I created all PRs in parallel and in none I could remove it on its own. But this is now the last one and I can do that here. Will update the PR. :)

Comment (pocmo) [reply components/feature/pwa/build.gradle:67]: Argh. Looks like there's an intent processor left that needs refactoring.

**Pull request merged**